README.md

```diff
@@ -105,10 +105,11 @@ $ tensorflowjs_converter \
 
 ### Web-friendly format
 
-The conversion script above produces 3 types of files:
+The conversion script above produces 4 types of files:
 
 * `tensorflowjs_model.pb` (the dataflow graph)
 * `weights_manifest.json` (weight manifest file)
+* `model.json` (the two above, in a single file)
```

Comment:
Thanks, you may want to identify that for Keras model, converter generates model.json and weight sharding file.
for other format it generates the other three.

Reply:
Agreed!